gargoyle/manager.py

```diff
@@ -40,6 +40,9 @@ def is_active(self, key, *instances, **kwargs):
 
         >>> gargoyle.is_active('my_feature', request) #doctest: +SKIP
         """
+        if hasattr(settings, 'GARGOYLE_OVERRIDE') and settings.GARGOYLE_OVERRIDE in (True, False, None,):
+            return settings.GARGOYLE_OVERRIDE
```

Comment:
Hmmm, I was thinking more like:

```python
override = getattr(settings, 'GARGOYLE_OVERRIDE', None)
if not None:
    return override
```
I don't think that "None" is a valid return for is_active.

Also, I'm thinking "GARGOYLE_GLOBAL_OVERRIDE" may be a tiny bit more explicit.